### PR TITLE
Missing link for "created a Kubernetes cluster with Jenkins X"

### PR DESCRIPTION
**Pull request by paulvi**
This is not actually PR, but bug as link is missing for "created a Kubernetes cluster with Jenkins X"

### CODE REVIEW

File: content/getting-started/next.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ toc: true
 
 OK so you have [got the jx CLI](/getting-started/install/) and you either
 
-* [created a Kubernetes cluster with Jenkins X](/getting-started/install-on-cluster/)
+* created a Kubernetes cluster with Jenkins X
```
Comment (LinuxSuRen) [inline content/getting-started/next.md]: Perhaps replace with a correct link instead of remove?

Comment (paulvi) [reply content/getting-started/next.md]: Yes, of course.

Somebody should know the correct link.  
I don't know as it is missing in the docs above, the link is actually for existing cluster, not how to get new

Comment (LinuxSuRen) [reply content/getting-started/next.md]: Should be https://jenkins-x.io/getting-started/create-cluster/.

Comment (paulvi) [reply content/getting-started/next.md]: see #1114

Comment (LinuxSuRen) [reply content/getting-started/next.md]: You have created a new PR. Current should be closed.

**Pull request closed without merging**